### ISSUE TITLE
loginctl enable-linger does not work for unprivileged user (without admin prompt)

### ISSUE THREAD

**Issue by mbiebl** [systemd/systemd#3365]
Following the v230 announcment on the fdo mailing list, I tried

```
$ loginctl enable-linger 
Could not enable linger: Access denied
```

@grawity mentioned that the following policy rule:

```
                <defaults>
                        <allow_any>yes</allow_any>
                </defaults>
```

might need an eplicit rule for allow_active and allow_inactive. Adding that did indeed fix the issue for me:

```
                <defaults>
                        <allow_any>yes</allow_any>
                        <allow_inactive>yes</allow_inactive>
                        <allow_active>yes</allow_active>
                </defaults>
```

```
$ loginctl enable-linger && echo success
success
```

**Comment by poettering** [comment]
Any chance you can post a PR for the change above? Would be much appreciated!

**Comment by poettering** [comment]
Fixed by #3373
